freedomofpress/securedrop-client
# [WIP] Polish offline UI to match mockup

WIP: 
- ~does not decrease opacity of left pane logo when starting out in offline mode~
- tweak gradient

# Description
- reverse gradient in top panel of main window (#583 for explanation)
- change sd logo opacity based on offline/online mode (#583 for explanation)
- small fix #585
- ~small fix #584~ (it may be better to leave this for another PR)

Fixes #583, #585, ~#584~


## Visual changes summary

SD logo on left panel now changes according to online/offline mode:

online: ![top0](https://user-images.githubusercontent.com/47065258/67465124-1dc9af80-f634-11e9-9228-a164089bc014.png)
offline: ![top1](https://user-images.githubusercontent.com/47065258/67465139-228e6380-f634-11e9-80ce-42fae1d82b4a.png)


Gradient on top panel now changes according to online/offline mode:

| online | offline |
|---------|---------|
|![commit-pane-online](https://user-images.githubusercontent.com/47065258/67463302-b3fbd680-f630-11e9-9f7e-db05b6dcdd76.png) | ![commit-pane-offline](https://user-images.githubusercontent.com/47065258/67463310-b5c59a00-f630-11e9-848e-9c2fd277e060.png)|

**note:** on the offline left pane the *sign in* button is cropped. That is #584 which is left unaddressed in this PR

# Test Plan
1. open in online mode
2. confirm the logo on the left pane has high opacity
3. confirm the top pane has right gradient for online mode
4. logout
5. confirm the logo on the left pane has now low opacity
6. confirm the top pane has [right gradient](https://zpl.io/Vqv0vwJ) for offline mode

# Checklist
If these changes modify code paths involving cryptography, the opening of files in VMs or network (via the RPC service) traffic, Qubes testing in the staging environment is required. For fine tuning of the graphical user interface, testing in any environment in Qubes is required. Please check as applicable:
